src/components/Modal/style.css

```diff
@@ -47,7 +48,6 @@
   background-color: white;
   padding: 16px;
   border-radius: 0 0 5px 5px;
-  overflow-y: auto;
```

Comment:
this one needs to stay

Reply:
hmmm when i had it that way before my change didn't work ugh

Reply:
o idk that was so that in the recipes modal the scrolling would keep the header fixed and only scroll inside the content so as long as both work

Reply:
header is still fixed with this